src/foam/u2/IFrameHTMLView.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2021 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.CLASS({
+  package: 'foam.u2',
+  name: 'IFrameHTMLView',
+  extends: 'foam.u2.HTMLElement',
```

Comment:
What happens if this just extends foam.u2.Element? Or extends foam.u2.View, and then you don't need to add the 'data' property?

Reply:
yeah View works, I didnt realize validator override wasnt needed since this isn't using add, thanks